service/src/main/java/bio/terra/cbas/common/MethodUtil.java

```diff
@@ -35,14 +41,48 @@ public static String convertGithubToRawUrl(String originalUrl)
   public static String convertToRawUrl(
```

Comment:
The reason I separated these is because, in `postMethod`, it seems like we need to pass the method URL to cromwell's `describe` endpoint before we actually create the `methodVersion` object (and the associated `githubMethodDetails` object). 

Right now, `getRawUrl` requires an existing `methodVersion`. I'm not saying the code can't be refactored to accomplish what you're asking for, but at the time, it wasn't obvious to me how to do it